packages/material-ui/src/styles/createTheme.d.ts

```diff
@@ -1,43 +1,20 @@
-import { Breakpoints, BreakpointsOptions } from './createBreakpoints';
-import { Mixins, MixinsOptions } from './createMixins';
+import { ThemeOptions as SystemThemeOptions, Theme as SystemTheme } from '@material-ui/system';
 import { Palette, PaletteOptions } from './createPalette';
 import { Typography, TypographyOptions } from './createTypography';
-import { Shadows } from './shadows';
-import { Shape, ShapeOptions } from './shape';
-import { Spacing, SpacingOptions } from './createSpacing';
-import { Transitions, TransitionsOptions } from './createTransitions';
 import { ZIndex, ZIndexOptions } from './zIndex';
 import { Components } from './components';
 
-export { Breakpoint, BreakpointOverrides } from './createBreakpoints';
-
-export type Direction = 'ltr' | 'rtl';
-
-export interface ThemeOptions {
```

Comment:
Part of these types are already included in the @material-ui/system `ThemeOptions` & `Theme`. We are now extending those.